docs/modules/publisher-directmethods.md

```diff
@@ -65,46 +61,48 @@ The `_V1` direct methods  uses the  payload schema as described below:
-
-_Note_: `OpcNodes` field is mandatory for PublishNodes_V1, UnpublishNodes_V1 and AddOrUpdateEndpoints_V1. It should not be specified for the rest of the direct methods.
+| Attribute                           | Mandatory | Type    | Default                   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| ----------------------------------- | --------- | ------- | ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `EndpointUrl`                       | yes       | String  | N/A                       | The OPC UA server endpoint URL                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `UseSecurity`                       | no        | Boolean | false                     | Controls whether to use a secure OPC UA mode to establish a session to the OPC UA server endpoint                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
>Controls whether to use a secure OPC UA mode to establish a session to the OPC UA server endpoint [](http://example.com/codeflow?start=90&length=97)

Not really related: I saw that the code does use Security.Best and Security.None. Is UseSecurity = false forcing Security.None and is Security.None enforcing SecurityProfile "None" or is it enforcing the lowest common SecurityProfile between client and server?

Reply:
I believe UseSecurity=False means "lowest" enforced by the server.

Reply:
@cristipogacean i have updated this to reflect that even "UseSecurity" == false may result in a secure connection.